components/kyma-environment-broker/internal/metrics/operation_result.go

```diff
@@ -180,11 +180,13 @@ func (c *OperationResultCollector) OnProvisioningStepProcessed(ctx context.Conte
 
 	var resultValue float64
 	switch stepProcessed.Operation.State {
-	case domain.InProgress:
+	case domain.InProgress, Pending, Retrying:
```

Comment:
I would dare to say that `InProgress` != `Pending`, consider expanding the switch statement to use the full range
https://github.com/kyma-project/control-plane/blob/8a652e3dadfc0b41c96e0ffa2470f8681a4d468c/components/kyma-environment-broker/internal/metrics/operation_result.go#L16-L23
like it's done in 
https://github.com/kyma-project/control-plane/blob/8a652e3dadfc0b41c96e0ffa2470f8681a4d468c/components/kyma-environment-broker/internal/metrics/operation_result.go#L136-L150